index.html

```diff
@@ -438,105 +438,90 @@ <h3 class="text-center">For more details, click the link below!</h3>
   <div class="container">
     <div class="row">
       <div class="section-title">
-        <h2>Already using Sugar? We have many activities for you!</h2>
-        <p>The Sugar Learning Platform is a complete environment for teaching and learning, which includes individual activities. If you're already using the Sugar Desktop Environment, then you can install from the activities below, which has links to some of our most popular activities.</p>
+        <h2>Explore Sugar Activities</h2>
+        <p>Discover a world of interactive learning with our most popular Sugar activities. Perfect for students already using the Sugar Desktop Environment.</p>
       </div>
     </div>
-    <div class="row "><!-- This is one too many for this 4x2 grid and the SVG -- and only this SVG -- renders too large
+    <div class="row activity-grid">
       <div class="col-sm-6 col-md-3">
```

Comment:
I experimented with changing these classes to be "col-md-4", like the section above.

![Screenshot from 2024-10-21 09-13-56](https://github.com/user-attachments/assets/b71a926f-f9b6-4b88-b885-e131ebf38d7e)

That seems to work better for using the full width of the screen.

WDYT?

Reply:
That looks good. I'll review and implement it in my next PR.

Reply:
@pikurasa How about using `col-sm-6`, `col-md-4`, and `col-lg-3`?

Reply:
Two columns for a large window screen will take up a lot of space, so making it 3 columns is wiser. For tablet screens, 2 columns seem perfect.

Reply:
That makes sense, but I'm pointing out something else: When it moves to one column, it doesn't use the full width.

Notice, in my screenshot above, the difference between "numbers" section and "finance".

I think, when it moves to one column, it should always fill the full width.

Reply:
Understood